lib/camp.js

```diff
@@ -629,13 +629,16 @@ function respondWithFile(req, res, path, ifNoFile) {
     }
     res.mime(p.extname(realpath).slice(1));
 
-    // Cache management.
-    if (+(new Date(req.headers['if-modified-since'])) >= +stats.mtime) {
-      res.statusCode = 304;
-      res.end();  // not modified.
+    // Cache management (compare timestamps at second-level precision).
+    var modified = new Date(stats.mtime);
```

Comment:
Sure! However, the `new Date(stats.mtime)` is there to make a copy of the `stats.mtime` Date object that we can modify (we could do `stats.mtime.setMilliseconds(0)` directly, but that would be less clean).